stdlib/src/builtin/simd.mojo

```diff
@@ -1142,6 +1142,18 @@ struct SIMD[type: DType, size: Int = simdwidthof[type]()](
         """
         return llvm_intrinsic["llvm.round", Self, has_side_effect=False](self)
 
+    @always_inline("nodebug")
+    fn __round__(self, ndigits: Int) -> Self:
```

Comment:
non-blocking: SIMD might benefit from having a separate overload that does this elementwise between two vectors. Might get tricky for the different dtypes, though. Anyway, just a followup idea.